tests/test_files.py

```diff
@@ -413,13 +413,30 @@ def test_find_python_files(self):
         self.make_file("sub/ssub/~s.py")            # nope: editor effluvia
         self.make_file("sub/lab/exp.py")            # nope: no __init__.py
         self.make_file("sub/windows.pyw")
-        py_files = set(find_python_files("sub"))
+        py_files = set(find_python_files("sub", False))
```

Comment:
Let's use the name of the argument here and below: `find_python_files("sub", include_namespace_packages=False)`
(hmm, that is a pretty long name.... :)  )

Reply:
done